src/index.ts

```diff
@@ -0,0 +1,12 @@
+import {Web3AuthLoginProvider, Web3AuthNetworkType} from './web3auth/types'
+import web3AuthProvider from './web3auth'
+import {
+  web3AuthProviderMetadata,
+} from './constants'
+import {Web3AuthConnection} from './web3auth/connection'
+
+type ConfigArgs = {clientId: string; networkType: Web3AuthNetworkType}
```

Comment:
I'd call the `networkType` key just `network`, the `type` prefix seems to me a bit redundant for the sake of understanding